frontegg/baseConfig/frontegg_proxy.py

```diff
@@ -62,12 +66,24 @@ def clean_headers(self, headers: dict, host: str) -> dict:
         new_headers[frontegg_headers['vendor_host']] = host
         return new_headers
 
+    def clean_response_headers(self, headers: dict) -> dict:
+        new_headers = dict()
+
+        ignored_headers = ['access-control-allow-credentials',
+                           'access-control-allow-origin']
+
+        for key, value in headers.items():
+            if key.lower() not in ignored_headers:
+                new_headers[key] = value
+
+        return new_headers
+
     def set_context(self, headers: dict, request) -> dict:
         context = self.context_callback(request)
-        if context.get('tenant_id'):
-            headers[frontegg_headers['tenant_id']] = context['tenant_id']
-        if context.get('user_id'):
-            headers[frontegg_headers['user_id']] = context['user_id']
+        if context.tenant_id:
```

Comment:
this will throw error if context.tenant_id does not exist. but context.get('tenant_id') won't throw error

Reply:
It will not throw as I changed it to return an object and not dict